documentation/chatmeleon-swagger.yaml

```diff
@@ -283,6 +284,208 @@ paths:
                   type: string
                   format: uri
 
+  /friend/send:
+    post:
+      description: Create a friend sending and pending relationship between two users.
+      tags:
+        - relationships
+      summary: Create a relationship.
+      security:
+        - bearerAuth: []
+      responses:
+        "200":
+          description: Relationship created successfully.
+        "401":
+          description: Unauthorized - JWT token is missing or invalid.
+        "403":
+          description: Unauthorized - User is unauthorized.
+
+      requestBody:
+        description: The ID of the other user to create relationship.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                relatedUserId:
+                  type: string
+                  pattern: "^[0-9a-f]{24}$"
+
+  /friend/accept:
+    put:
+      description: Create a friend relationship between two users.
+      tags:
+        - relationships
+      summary: Create a friend relationship
+      security:
+        - bearerAuth: []
+      responses:
+        "200":
+          description: Relationship created successfully.
+        "401":
+          description: Unauthorized - JWT token is missing or invalid.
+        "403":
+          description: Unauthorized - User is unauthorized.
+
+      requestBody:
+        description: The ID of the other user to create relationship.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                relatedUserId:
+                  type: string
+                  pattern: "^[0-9a-f]{24}$"
+
+  /friend/decline:
```

Comment:
Same as above:
- What if this is called more than once?
- What if `decline` endpoint is called before a friend request is made?
- What if `decline` endpoint is called after the friend request is accepted?